metricbeat/docs/metricbeat-options.asciidoc

```diff
@@ -45,90 +100,107 @@ metricbeat.modules:
   hosts: ["root@tcp(127.0.0.1:3306)/"]
 ------------------------------------------------------------------------------
 
-[float]
-== Configuration combinations
-
-You can specify a configuration that uses different combinations of modules, metricsets, periods, and hosts. In the following example, the Redis host is crawled for `stats` information every second because this is critical data, but the full list of Apache metricsets is only fetched every 30 seconds because the metrics are less critical.
+In the following example, the Redis host is crawled for `stats` information
+every second because this is critical data, but the full list of Apache
+metricsets is only fetched every 30 seconds because the metrics are less
+critical.
 
 [source,yaml]
 ----
-metricbeat:
-  modules:
-    - module: redis
-      metricsets: ["info"]
-      hosts: ["host1"]
-      period: 1s
-      enabled: true
-    - module: apache
-      metricsets: ["info"]
-      hosts: ["host1"]
-      period: 30s
-      enabled: true
+metricbeat.modules:
```

Comment:
is this line still wanted? I'm not sure from the context

Reply:
yah, because this sections shows how to configure multiple modules in the metricbeat.yml file. I don't want users to get confused and add this to one of the config files under modules.d.